### PR TITLE
job-level port mappings

### PR DESCRIPTION
**Pull request by sshookman**
[#147]

### Solution
Adds port mappings at the job level and concatenates them to the global-level port mappings. Both default to an empty array, so neither port mappings are needed, but each (or both) can be used.

### CODE REVIEW

**Comment by sshookman** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- skelebot/objects/job.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/skelebot/skelebot/pullRequest?prid=5154635&bid=16928653)

File: skelebot/systems/execution/executor.py [hunk 1]
```diff
@@ -67,7 +67,8 @@ def executeJob(config, args, job):
     else:
         if (not args.skip_build_global):
             buildDocker(config)
-        status = runDocker(config, command, job.mode, config.ports, job.mappings, job.name)
+        ports = config.ports + job.ports
```
Comment (jagmoreira) [inline skelebot/systems/execution/executor.py]: Maybe remove duplicates here, in case users have the same port mapping in multiple places

**Review by jagmoreira — CHANGES_REQUESTED** [review]

File: skelebot/systems/execution/executor.py (hunk 1 above)
Comment (sshookman) [reply skelebot/systems/execution/executor.py]: Updated

**Comment by sshookman** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- skelebot/objects/job.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/skelebot/skelebot/pullRequest?prid=5154635&bid=16928653)

**Review by jagmoreira — APPROVED** [review]

**Pull request merged**